RFCs/RFC-001: Introspect streams.md

```diff
@@ -0,0 +1,62 @@
+# RFC-001: Introspect Streams
+
+**To be Reviewed By:** 11/19/2019
+
+**Authors:** Swapnil Bawaskar
+
+**Status:** **Active** | Accepted | Dropped | Superseded
+
+**Superseded by:** N/A
+
+**Related:** N/A
+
+
+## Problem
+When the streaming runtime is installed, riff users can create streams (backed by a stream provider e.g. kafka). Users can write functions that read from and write to these streams,
+but there is no easy mechanism to get events into the stream and look into the contents of the stream which are helpful while developing/demoing.
+
+### Anti-Goals
+We will only address this problem for development/demos, not production, so topics like auth/authz are out of scope for this document.
+
+## Solution
+### Ideal solution:
+The  is to provide users with only one cli (riff cli) that will enable users to send/receive messages to/from a stream (commands below). We will have to ask the users to port-forward the gateway service if they are running outside the cluster.
```

Comment:
In the "ideal solution", why would the user have to manually do the complicated step of portforwarding? The ideal solution is seamless

Reply:
Sure, we could setup port forwarding automatically, but I think that should be its own rfc which should also tackle the deployers from core and knative runtime.